tests/test_filtsmooth/test_statespace/conftest.py

```diff
@@ -0,0 +1,98 @@
+"""Fixtures to be shared across all modules in this directory.
```

Comment:
The filename is not very clear to me

Reply:
It is a pytest thing, not my decision :(